spec/request/v0/higher_level_reviews_controller_request_spec.rb

```diff
@@ -24,6 +24,8 @@ def personal_information_logs
       VCR.use_cassette('decision_review/HLR-CREATE-RESPONSE-200') do
         subject
         expect(response).to be_successful
+        parsed_response = JSON.parse(response.body)
+        expect(parsed_response['data']['id']).to eq(AppealSubmission.last.submitted_appeal_uuid)
```

Comment:
if we wanted to avoid using `last` we could do something more like:
```
        expect(AppealSubmission.all.count).to eq 0
        subject
        expect(response).to be_successful
        parsed_response = JSON.parse(response.body)
        expect(AppealSubmission.where(submitted_appeal_id: parsed_response['data']['id'])).to eq 1
```
i've seen reliance on DB ordering cause tests to become flaky. I don't _think_ that's much of a problem in vets-api 🤷